.changes/README.md

```diff
@@ -0,0 +1,23 @@
+# Changes
+
+This stores changes to automatically generate the changelog, to avoid merge conflicts. Files should be in a JSON format, with the following format:
+
+```json
+{
+    "description": "single-line description to add to the CHANGELOG.",
+    "issues": [894],
+    "type": "added",
+    "breaking": false
+}
+```
```

Comment:
can we add a `template.json` file with this, probably without the optional breaking

Reply:
Sure, guess we just ignore it?

Reply:
Done, I've added 2 files, one for an array and one for an object-based change. I'll add them to the README as well.